client/components/payment-methods-checkboxes/payment-method-checkbox.tsx

```diff
@@ -22,10 +22,37 @@ import PaymentMethodsMap from '../../payment-methods-map';
 import Pill from '../pill';
 import { HoverTooltip } from 'components/tooltip';
 import './payment-method-checkbox.scss';
-import { useManualCapture } from 'wcpay/data';
+import { useManualCapture, useAccountDefaultCurrency } from 'wcpay/data';
+import { FeeStructure } from 'wcpay/types/fees';
+
+type PaymentMethodProps = {
+	name: string;
+};
+
+const getDescription = ( name: string, currency: string ) => {
+	const paymentMethod = PaymentMethodsMap[ name ];
+
+	if ( ! paymentMethod ) return null;
+
+	let description = paymentMethod.description;
+	const isAllowingPayLater = paymentMethod.allows_pay_later;
+
+	if ( isAllowingPayLater ) {
+		description = sprintf( description, currency.toUpperCase() );
+	}
+
+	return description;
+};
+
+const PaymentMethodDescription: React.FC< PaymentMethodProps > = ( {
+	name,
+} ) => {
+	const [ stripeAccountDefaultCurrency ] = useAccountDefaultCurrency();
+	const description = useMemo(
```

Comment:
Nice! Though all descriptions will be unique, and cashing here has no practical value.

Reply:
Indeed, after a more careful read of the docs, the recommendation is to only use memo when there are expensive calculations involved. Since this is not the case, I'm removing it in this commit [e4f8971](https://github.com/Automattic/woocommerce-payments/pull/6880/commits/e4f8971d33dcf8b99c726edb2115244362072666)

Thanks!